src/display_manager.py

```diff
@@ -38,8 +40,8 @@ def __init__(self, phy):
         s.phy = phy
         s.screen = None
         s.pause = 0
-        s.width, s.height = 0, 0
-        s.x, s.y = 0, 0
+        s.width, s.height = Vector(0, 0)
```

Comment:
I'm not 100% sure I follow why you're using the vector class like this. Could you please explain?